sdk/textanalytics/Azure.AI.TextAnalytics/ToStringGuidence.md

```diff
@@ -0,0 +1,30 @@
+# ToString
+
+## Considerations
+- Most useful / main pain points
+- Following guidance from BCL team and .NET team members
+- Applied to TA and FR
+- No added logic into ToString. Plain and simple.
+
+## General Guidance
```

Comment:
These are not intended to be consumed/added to generator.
We need more info to see the cases that apply to all libraries and will not cause problems

Reply:
have a list of properties? or a extension for swagger?
is it worth it though having ToString as a simple thing to implement...